src/App.jsx

```diff
@@ -77,9 +79,9 @@ export function App() {
       if (!response.ok) {
         throw new Error('Network response was not ok');
       }
-      return response.blob();
+      return response.text();
```

Comment:
I reckon that should be blob, not text.

also headers can go, this is not needed
```js
headers: { 'Content-Type': 'application/json' },```
we are not adding any json to the request

Reply:
Reverted `blob()`.
Removed headers.